native/src/main/java/io/ballerina/persist/nodegenerator/BalFileConstants.java

```diff
@@ -0,0 +1,118 @@
+            "('object, filter);";
+    public static final String DELETE_RUN_DELETE_QUERY = "_ = check self.persistClient.runDeleteQuery(filter);";
+    public static final String CLOSE_PERSIST_CLIENT = "return self.persistClient.close();";
+    public static final String INIT_STREAM_STATEMENT = "self.anydataStream = anydataStream;";
+    public static final String NEXT_STREAM_RETURN_TYPE = "record {|%s value;|}|error?";
+    public static final String NEXT_STREAM_STREAM_VALUE = "var streamValue = self.anydataStream.next();";
+
+    public static final String NEXT_STREAM_IF_STATEMENT = "streamValue is ()";
+    public static final String NEXT_STREAM_ELSE_IF_STATEMENT = "(streamValue is error)";
+    public static final String NEXT_STREAM_RETURN_STREAM_VALUE = "return streamValue;";
+    public static final String NEXT_STREAM_ELSE_STATEMENT = "record {|%s value;|} nextRecord = " +
+            "{value: check streamValue.value.cloneWithType(%s)};";
+    public static final String CLOSE_STREAM_STATEMENT = "return self.anydataStream.close();";
+
+    public static final String CONFIGURABLE_PORT = "configurable int port = ?;";
+    public static final String CONFIGURABLE_HOST = "configurable string host = ?;";
+    public static final String CONFIGURABLE_USER = "configurable string user = ?;";
+    public static final String CONFIGURABLE_PASSWORD = "configurable string password = ?;";
+    public static final String CONFIGURABLE_DATABASE = "configurable string database = ?;";
+    public static final String INIT = "init";
+    public static final String READ = "read";
+    public static final String CREATE = "create";
+    public static final String DELETE = "delete";
+    public static final String READ_BY_KEY = "readByKey";
+    public static final String UPDATE = "update";
+    public static final String CLOSE = "close";
+    public static final String NEXT = "next";
+
+    public static final String RECORD_FIELD_VAR = "%s:%s";
+    public static final String RECORD_FIELD_LAST_INSERT_ID = "%s: <%s>result.lastInsertId";
+    public static final String RECORD_FIELD_VALUE = "%s: value.%s";
+    public static final String KEYWORD_BALLERINA = "ballerina";
+    public static final String KEYWORD_BALLERINAX = "ballerinax";
+    public static final String KEYWORD_PRIVATE = "private";
+    public static final String KEYWORD_CLIENT = "client";
+    public static final String KEYWORD_PUBLIC = "public";
+    public static final String KEYWORD_ISOLATED = "isolated";
+    public static final String KEYWORD_FINAL = "final";
+    public static final String KEYWORD_REMOTE = "remote";
+    public static final String KEYWORD_INT = "int";
+    public static final String KEYWORD_STRING = "string";
+    public static final String KEYWORD_SQL = "sql";
+    public static final String KEYWORD_MYSQL = "mysql";
+    public static final String KEYWORD_TIME = "time";
+
+
+
+
+
```

Comment:
```suggestion
```

Reply:
Updated in [d6a53d3](https://github.com/ballerina-platform/persist-tools/pull/15/commits/d6a53d3f948c8f076acb3cc1f2730258a998cf40)